system/ee/installer/controllers/wizard.php

```diff
@@ -33,9 +33,11 @@ class Wizard extends CI_Controller {
 	public $header            = '';
 	public $subtitle          = '';
 
-	private $current_step = 1;
-	private $steps        = 3;
-	private $addon_step   = FALSE;
+	private $current_step 			= 1;
+	private $steps        			= 3;
+	private $addon_step   			= FALSE;
+	private $shouldBackupDatabase	= false;
+	private $shouldUpgradeAddons 	= false;
```

Comment:
Constant's case should match here.

Reply:
@obfuscode With the move to PSR-2, are we going lowercase or uppercase?

Reply:
I'd say user uppercase for now as we're not refactoring existing code and mixing the two is less desirable.

Reply:
@obfuscode fixed on latest commit.